thirtyinch/src/main/java/net/grandcentrix/thirtyinch/TiConfiguration.java

```diff
@@ -198,6 +199,16 @@ public Builder setUseStaticSaviorToRetain(final boolean enabled) {
             return this;
         }
 
+        /**
+         * An {@link TiPresenterSerializer} instance should be provided, if the {@link TiPresenter} should
+         * be restored, after a process has died, but the UI component is being recreated. Since this is
+         * a rare use case the default value is {@code null} and presenters aren't restored.
+         * @param serializer An implementation which can serialize and deserialize any presenter.
+         */
+        public Builder setPresenterSerializer(TiPresenterSerializer serializer) {
```

Comment:
That was the ideal place for me. It's pretty handy with the global config and optional for tests.